pages/common/pdfjam.md

```diff
@@ -1,9 +1,9 @@
 # pdfjam
 
-> Shell frontend for the LaTeX pdfpages package.
+> Shell frontend for the LaTeX pdfpages package for mingling PDFs.
```

Comment:
Just out of interest: Does this command create a `.tex` file, in which it puts the code to get the desired output and compiles it? Does that mean you have to have a whole TeX distribution installed?